src/file.rs

```diff
@@ -16,13 +16,13 @@ use std::os::windows::fs::MetadataExt;
 
 /// General file information.
 #[derive(Debug)]
-pub struct FileInfo<'a> {
+pub struct FileInfo {
     /// Path of the file.
-    pub path: &'a str,
+    pub path: String,
     /// Arguments of the file.
-    pub arguments: Option<Vec<&'a str>>,
+    pub arguments: Option<Vec<String>>,
     /// Bytes of the file.
-    pub bytes: &'a [u8],
+    pub bytes: Box<[u8]>,
```

Comment:
Why do we need to remove the lifetimes here?

Reply:
Short answer - beacause now all application live in one loop in lib.rs (without recreating it with run function).

More detailed. Reference path: &'a str removed because when wee create a new analyzer in `Event::Restart` handler live time of analizer (and FileInfo in it) is greater than the lifetime of the variable `path` from the event. &'a [u8] changed to Box<[u8]> with same reason, file data will be freed but we still need it in an analyzer. I dont think that this is a big problem at all, Box<[u8]> still protects us from copy of the file data, and own file path instead of reference to it - not a big overhead.